azure-iot-device/azure/iot/device/constant.py

```diff
@@ -0,0 +1,12 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module defines constants for use across the azure-iot-device package
+"""
+
+VERSION = "2.0.0-preview.8"
+USER_AGENT = "py-azure-iot-device/{version}".format(version=VERSION)
+IOTHUB_API_VERSION = "2018-06-30"
```

Comment:
>IOTHUB_API_VERSION [](start = 0, length = 18)

To save renaming later, I think these are more accurately IOTHUB_DEVICE_API_VERSION and PROVISIONING_DEVICE_API_VERSION.  I don't know if device and service API versions update in lock-step, but I think it might be a mistake to assume that they do.

Reply:
Given that this is within the azure-iot-device package I figured the "device"ness was implied.

---
In reply to: [307826928](https://github.com/Azure/azure-iot-sdk-python-preview/pull/147#discussion_r307826928) [](ancestors = 307826928)